refinerytrace/tracer.go

```diff
@@ -0,0 +1,30 @@
+package refinerytrace
```

Comment:
I had deleted this ... i think it's no longer in use, so you should remove it from this PR too.